test/DuoApiTest.csproj

```diff
@@ -9,8 +9,8 @@
     <ProjectGuid>{6B97B9FB-E553-494C-BD50-4BF7DB5C2184}</ProjectGuid>
     <OutputType>Library</OutputType>
     <AppDesignerFolder>Properties</AppDesignerFolder>
-    <RootNamespace>test</RootNamespace>
-    <AssemblyName>test</AssemblyName>
+    <RootNamespace>DuoApiTest</RootNamespace>
+    <AssemblyName>DuoApiTest</AssemblyName>
```

Comment:
Renamed the assembly containing the test files from `test` to `DuoApiTest` to mirror the name of the solution/folder the tests are in.